docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-2.1-applications-linux-containers-aks-kubernetes.md

```diff
@@ -0,0 +1,180 @@
+
+Visual Studio will show a list with Templates for Web Projects. For our example you must select **Web API,** that means you will create an ASP.NET Web API Application.
+
+Verify that you have selected ASP.NET Core 2.1 as the framework. The .NET Core 2.1 is included in the last version of Visual Studio 2017 and is automatically installed and configured for you when you install Visual Studio 2017.
+
+![Visual Studio dialog for selecting the type of an ASP.NET Core Web Application with API option selected.](media/create-web-api-application.png)
+
+**Figure 4-37**. Selecting ASP.NET CORE 2.1 and Web API project type
+
+If you have any previous version of .NET Core, you can download and install the 2.1 version from [https://www.microsoft.com/net/download/core#/sdk](https://www.microsoft.com/net/download/core#/sdk)
+
+You can add Docker support when creating the project in the previous step, or later, so you can "Dockerize" your project any time. To add the Docker support after the project creation, right-click on the solution file and select **Add > Docker support** on the context menu.
+
+![Context menu option to add Docker support to a existing project: Right click (on the project) > Add > Docker Support.](media/add-docker-support-to-project.png)
+
+**Figure 4-38**. Adding Docker support to existing project
+
+To complete adding Docker support, you have the choice of Windows or Linux, in this case select **Linux**, because AKS doesn’t support Windows Containers (as of late 2018).
+
+![Option dialog to select Target OS for Dockerfile.](media/select-linux-docker-support.png)
+
+**Figure 4-39**. Selecting Linux Containers.
+
+With these simple steps, you will have your ASP.NET Core 2.1 application running on a Linux Container.
+
+As you can see the, integration between Visual Studio 2017 and Docker is totally oriented to the developer’s productivity.
+
+Now you can run your application with the key **F5** or using the Play Button
+
+After running the project, you can list the images using the `docker images` command, and you should see the `mssampleapplication` image created with the automatic deploy of our project with Visual Studio 2017.
+
+```console
+# docker images
+```
+
+![Console output from the docker images command, shows a list with: Repository, Tag, Image Id, Created (date), and Size.](media/docker-images-command.png)
+
+**Figure 4-40**. View of docker images
+
+## Register the Solution in the Azure Container Registry
+
+Now, we have to upload the image to any Docker registry, like [Azure Container Registry (ACR)](https://azure.microsoft.com/services/container-registry/) or Docker Hub so the images can be deployed to the AKS cluster from that registry. In this case, we’re uploading the image to Azure Container Registry.
+
+### Create the image in Release mode
+
+We'll now create the image in **Release** Mode (ready for production) changing to Release as shown here and running the application as did before.
```

Comment:
We'll now create -> Create 

running -> run

did -> you did

Reply:
done.